source/manual/common-aws-tasks-for-2nd-line-support.html.md

```diff
@@ -0,0 +1,232 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Common AWS tasks for 2nd line support
+section: AWS
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+---
+
+This document details some of the tasks that GOV.UK 2nd line support may
+carry out regarding AWS.
+
+## Getting help with AWS
+
+### Read the docs
+
+The [AWS docs](https://docs.aws.amazon.com/) are comprehensive, and the
+GOV.UK developer docs explain how to do common tasks.
+
+### Raise an AWS support request
+
+We pay AWS for Premium Support. You are strongly encouraged to contact AWS to
+help you solve problems when using AWS products. Contacting AWS Support is a
+very common procedure.
+
+See our documentation on [how to escalate to AWS support][].
+
+[how to escalate to AWS support]: https://docs.publishing.service.gov.uk/manual/how-to-escalate-to-AWS-support.html
+
+### Internal support
+
+2nd line support is the only support team responsible for investigating issues
+related to AWS. Site Reliability Engineers may be able to help you, but there
+is currently no support rota for SREs.
```

Comment:
Maybe we could reword this a bit... something like:

> Usually, 2nd line support should be able to investigate issues related to AWS.

> During working hours, one of the Site Reliability Engineers on the replatforming team may be able to provide advice or expert knowledge. Outside of working hours, you should escalate to AWS support if the engineers on call can't resolve an issue themselves.

Reply:
Thanks, I've made that change.